samples/OpenIdConnect.AzureAdSample/Startup.cs

```diff
@@ -70,7 +70,7 @@ public void ConfigureServices(IServiceCollection services)
                         var authContext = new AuthenticationContext(Authority, AuthPropertiesTokenCache.ForCodeRedemption(context.Properties));
 
                         var result = await authContext.AcquireTokenByAuthorizationCodeAsync(
-                            context.ProtocolMessage.Code, new Uri(currentUri), credential, Resource);
```

Comment:
If I really wanted to waste time convincing you, @HaoK, that would be for the context method names... :trollface: 

`ProtocolMessage` is not ideal, but at least, it's self-explanatory.